docs/gen-1/red-blue/main-glitchless/race-route/README.md

```diff
@@ -353,21 +356,22 @@ If you have less than 9 HP, Fight the Gentleman in [this room here](https://gunn
 
 ## Route 8
 Pick up the hidden Max Ether [here](https://gunnermaniac.com/pokeworld?map=1#336/97)
+
 **Gambler:**
 * Growlithe:
 	- If you have less than 5 HP Potion
 	- BB
 * Vulpix: Thrash
 
 ## Underground
-- Get the Hidden Elixer [here](http://gunnermaniac.com/pokeworld?map=121#21/5)
-	- Get the Hidden Nugget [here](http://gunnermaniac.com/pokeworld?map=121#12/2)
+- Get the hidden Elixer [here](http://gunnermaniac.com/pokeworld?map=121#21/5)
```

Comment:
perhaps link on the words [hidden Elixer] and drop "here" ? (same for next line)